### PR TITLE
Updated readme file

### PR DESCRIPTION
**Pull request by yotama9**
- Updated the documentation for the Shortcuts readme page
- changed the empty mode from 'disabled' to 'unchaged' in the shortcut documentations
- minor fix to the shortcut readme file
- updated the read.me file to include explanation about the welcome interface and the various elements of it
- Added screenshot of the introduction screen.

### CODE REVIEW

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:21]: there is a double space

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:24]: Maybe call it "Download the app version [...]"

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:27]: maybe opened? without the brackets its:
Once you've open the app

**Review by CodeDoctorDE — CHANGES_REQUESTED** [review]
Hi, I have some things I would change before merging.
Can you duplicate these changes also for the 2.0 version?
It's in docs/versioned_docs/2.0

File: docs/docs/README.md
Comment (yotama9) [reply docs/docs/README.md:27]: Hahah tried to keep it less "formal"

Comment (CodeDoctorDE) [reply docs/docs/README.md]: It's okay but maybe
"Once you've (installed and have) opened the app" or something like that to stay in the time form

**Comment by CodeDoctorDE** [comment]
Hi, and I cannot merge this since it has conflicts to the current develop branch. Can you resolve them?

File: docs/docs/README.md
Comment (Luciogi) [inline docs/docs/README.md:21]:
```suggestion
> You can use the sidebar to navigate quickly through the wiki.
```

Comment (CodeDoctorDE) [reply docs/docs/README.md:21]: Oh I didn't saw it while reviewing.
And please keep a line above and below free for better reading

File: docs/docs/shortcuts.md
Comment (Luciogi) [inline docs/docs/shortcuts.md:52]:
```suggestion
Here, 1 refers to the lasso tool, 2 to the pan tool etc.
```

File: docs/docs/README.md
Comment (Luciogi) [inline docs/docs/README.md:35]: this part is not rendering correctly
Fix it

Comment (Luciogi) [reply docs/docs/README.md:35]: I guess, it only renders on web(docs) not on github md preview

File: docs/docs/README.md
Comment (CodeDoctorDE) [reply docs/docs/README.md:12]: No, it's okay. It shouldn't be readable on github but if we generate it with docusaurus it should be correctly displayed

File: docs/docs/README.md
Comment (CodeDoctorDE) [reply docs/docs/README.md:35]: Yeah, maybe. I started the preview ci, we can look at it if it will be rendered correctly

File: docs/docs/README.md
Comment (Luciogi) [inline docs/docs/README.md:18]: Umm How is it ?
```suggestion
Butterfly is a note taking app that is aiming for unified experience across all platforms. It provides simple UI with powerful customization.
```

Comment (yotama9) [reply docs/docs/README.md:18]: I'm fine with both, @CodeDoctorDE what do you think is better describing the app?

Comment (CodeDoctorDE) [reply docs/docs/README.md:18]: Hmm I have no strong opinion on this. I think the minimalistic aspect is important but then you could fix the typo in `Butterfly is a note taking up`

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:86]: please don't use dots in headlines.
And I would call it Document view?

**Review by CodeDoctorDE — CHANGES_REQUESTED** [review]

**Comment by CodeDoctorDE** [comment]
Can you add syncing tab change to these two tabs?
https://docusaurus.io/docs/markdown-features/tabs#syncing-tab-choices

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:107]:
```suggestion
		By default this bar has the <ScribbleLoop/> lasso tool; the <Pen/> pen tool; the <Path/> path erase tool; the <ArrowCounterClockwise/> undo and <ArrowClockwise/> redo butons, and the <Hand/> hand tool. Then you have the <Plus/> button to add elements to the tool bar, the <Wrench/> button to configure the document, and finally the <Lock/> tool to look the zoom and/or the position of the screen. 
```

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:122]:
```suggestion
		By default this bar has the <ScribbleLoop/> lasso tool; the <Pen/> pen tool; the <Path/> path erase tool; the <ArrowCounterClockwise/> undo and <ArrowClockwise/> redo butons, and the <Hand/> hand tool. Then you have the <Plus/> button to add elements to the tool bar, the <Wrench/> button to configure the document, and finally the <Lock/> tool to look the zoom and/or the position of the screen. 
```

File: docs/docs/README.md
Comment (CodeDoctorDE) [inline docs/docs/README.md:125]:
```suggestion
		2. After selecting a tool, a small menu will appear to configure the it. Long clicking the tool again will allow you to drag it to a new position
```

**Review by CodeDoctorDE — CHANGES_REQUESTED** [review]
Otherwise it looks good

**Comment by CodeDoctorDE** [comment]
Thanks for contributing!

**Pull request merged**
